src/casp/ml/data/transforms.py

```diff
@@ -0,0 +1,57 @@
+import typing as t
+
+import torch
+
+from casp.ml.running_stats import OnePassMeanVarStd
+from casp.ml.utils import PickleMixin
+
+
+class CASTransform(PickleMixin):
+    def __call__(self, tensor: torch.Tensor) -> torch.Tensor:
+        raise NotImplementedError
+
+
+class Compose(CASTransform):
+    """
+    Inspired by `torchvision.transforms.Compose`
```

Comment:
Yeah, we sure can, but it's a part of `torchvision` module which is made for image recognition. I thought it would be confusing to use. That was the only reason.

Reply:
Also, I didn't want to install the whole package just for these 20 lines of code.